data/hai/hai reveal 1 intro.txt

```diff
@@ -729,19 +730,19 @@ mission "Hai Leaks Response 4"
 		has "Hai Leaks Response 3: done"
 	on offer
 		conversation
-			`When you land on Allhome, you are greeted by the Hai Captain of the local forces. You're not an expert at judging Hai ages, but he appears to be a little younger than expected.`
+			`When you land on Allhome, you are greeted by the quartermaster of the local Hai security forces. You're not an expert at judging Hai ages, but he appears to be a little younger than expected.`
 			`	"Captain <last>! I heard you were coming to assist."`
 			choice
 				`	"Time to rout some pirates, baby!"`
 				`	"I'm here to reinforce you. What can I do?"`
 			`	"Uh well, yes. I presume you saw the pirates on your way in? They've been appearing en masse and overwhelming security forces to raid technology and goods from the Hai traders and their security. Some of them have even been sneaking down onto the planet, capturing Hai, and taking them away."`
 			choice
 				`	"I have heard about the kidnapping."`
-			`	He nods. "Unfortunately, it's taking a while for them to redistribute an appropriately resourced defense force, thanks to the Unfettered. It's bad enough that we've had to bring the old Ladybug models out of mothballs to bolster our forces here."`
+			`	He nods. "Unfortunately, it's taking a while for them to redistribute an appropriately resourced defense force, because of the Unfettered. It's bad enough that we've had to reinstate these old Ladybug models to bolster our forces here."`
```

Comment:
```suggestion
			`	He nods. "Unfortunately, it's taking a while for them to redistribute an appropriately resourced defense force, because of the Unfettered. It's bad enough that we've had to bring these old Ladybug models out of mothballs to bolster our forces here."`
```

I like the "the"->"these," change but think the less formal phrasing better demonstrates the quartermaster's frustration.